content/en/events/banners/kubernetes-1.22-with-older-releases.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Kubernetes 1.22 Release statement
+period_start: 2021-08-04
+period_duration: 180
+max_impressions: 20
+timeout: 120
+link: /docs/releases/supported-releases#support-status-of-istio-releases
+
+---
+
+Warning! Kubernetes 1.22 is not supported by Istio release 1.9.x. Do not upgrade your Kubernetes version if you are using Istio versions prior to 1.10.
```

Comment:
Is this better? 

```suggestion
Istio release 1.9.x is not compatible with Kubernetes 1.22. If you upgrade your Kubernetes cluster, please consider upgrading to Istio 1.10 or newer. 
```

Reply:
I'm not sure `consider` is accurate, at least if they are upgrading to 1.22.